mlhaiti/api/urls.py

```diff
@@ -1,13 +1,8 @@
-from django.urls import path
-from . import views
+from django.urls import path, include
 
-urlpatterns = [
-	# USER END-POINTS
-    path('users', views.UserList.as_view()),
-    path('user/list', views.UserList.as_view()),
+from mlhaiti.accounts.api import views as  account_views
 
-    path('forums', views.ForumList.as_view()),    
-    path('forum/list', views.ForumList.as_view()),    
-    # path('forum/<str:code>/messages', views.ForumMessagesList.as_view()),
-    # path('forum/<str:code>/message/list', views.ForumMessagesList.as_view()),
+urlpatterns = [
+    # ACCOUNT END-POINTS
```

Comment:
Unless I misunderstood the logic, I think we need to have an endpoint to list users, and forums.
No need to remove them I think

Reply:
Well you shoud add them as user stories first. Then we implement them. Every pull request solve at least one user stories. When you add the feature as user story first you allow the rest the team to accept it or reject it. This bring knowledge accross the project

Reply:
Correct!

Reply:
:+1: